.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+oshop-dashboard/node_modules
```

Comment:
i went through this with alex yesterday so he can probably help with this. We should have all the contents of oshop-dashboard in the root directory